apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -0,0 +1,584 @@
+                                >
+                                  <i class="fa fa-edit"></i>
+                                  <span>Edit</span>
+                                </a>
+                              `
+                            : html`info.json`}
+                        `
+                      : ''}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Type</th>
+                  <td>${resLocals.question.type}</td>
+                </tr>
+                <tr>
+                  <th>Topic</th>
+                  <td>
+                    ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                      topic: resLocals.topic,
+                    })}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Tags</th>
+                  <td>
+                    ${renderEjs(__filename, "<%- include('../partials/tags') %>", {
+                      tags: resLocals.tags,
+                    })}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Issues</th>
+                  <td>
+                    ${renderEjs(__filename, "<%- include('../partials/issueBadge') %>", {
+                      count: resLocals.open_issue_count,
+                      issueQid: resLocals.question.qid,
+                      suppressLink: resLocals.suppressLink,
+                      urlPrefix: resLocals.urlPrefix,
+                    })}
+                  </td>
+                </tr>
+                <tr>
+                  <th>Assessments</th>
+                  <td>
+                    ${resLocals.assessments
+                      ? renderEjs(__filename, "<%- include('../partials/assessments') %>", {
+                          assessments: resLocals.assessments,
+                          urlPrefix: resLocals.urlPrefix,
+                        })
+                      : ''}
+                  </td>
+                </tr>
+                ${sharingEnabled
+                  ? html`
+                      <tr>
+                        <th class="align-middle">Sharing</th>
+                        <td data-testid="shared-with">
+                          ${resLocals.question.shared_publicly
+                            ? html`<div class="badge color-green3">Public</div>
+                                This question is publicly shared.`
+                            : html`
+                                ${sharingSetsIn.length === 0
+                                  ? html`Not Shared`
+                                  : html`
+                                      Shared With:
+                                      ${sharingSetsIn.map(function (sharing_set) {
+                                        return html`<span class="badge color-gray1"
+                                          >${sharing_set?.name}</span
+                                        >`;
```

Comment:
Usual comment about splitting this into multiple lines.

Reply:
This is now split out to multiple lines (now lines 220-224) in [e8cc26c](https://github.com/PrairieLearn/PrairieLearn/pull/9571/commits/e8cc26c79af51815e4e9bdbf5d82f9f432f5decc).